documents/state_machine.py

```diff
@@ -0,0 +1,54 @@
+from config import config
+from models.db import db
+from models.User import User
+import jwt
+from models.Document import Document
+import utils
+
+@utils.server_error_check
```

Comment:
comment for this statement everywhere in this file: why is it required? The API's try catch will take care of this no?